client/components/checkbox/index.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+import classNames from 'classnames';
+import hasOwn from 'lodash/object/has';
+
+/**
+* Interal dependencies
+*/
+import Gridicon from 'components/gridicon';
+
+module.exports = React.createClass( {
+	displayName: 'Checkbox',
+
+	propTypes: {
+		onChange: React.PropTypes.func,
+		checked: React.PropTypes.any
```

Comment:
we actually specify a boolean here in the documentation, so we should match that here…

```js
import React, { PropTypes } from 'react';

propTypes: {
	checked: PropTypes.bool.isRequired,
	disabled: PropTypes.bool,
	onChange: PropTypes.func
}
```